### PR TITLE
Make all instances of doc links in logs clickable

### PR DESCRIPTION
**Pull request by angelapwen**
Fixes https://github.com/github/codeql-action/issues/2354. 

This PR updates all log messages with links to GitHub docs so that they are not succeeded by punctuation marks, in order to make sure the links are clickable from our logs. 

### Merge / deployment checklist

- [x] Confirm this change is backwards compatible with existing workflows.
- [x] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [x] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

File: src/codeql.test.ts [hunk 1]
```diff
@@ -912,8 +912,8 @@ test("runTool summarizes autobuilder errors", async (t) => {
       instanceOf: CommandInvocationError,
       message:
         "We were unable to automatically build your code. Please provide manual build steps. " +
-        "For more information, see " +
-        "https://docs.github.com/en/code-security/code-scanning/troubleshooting-code-scanning/automatic-build-failed. " +
+        "See https://docs.github.com/en/code-security/code-scanning/troubleshooting-code-scanning/automatic-build-failed " +
```
Comment (NlightNFotis) [inline src/codeql.test.ts:915]: I'm wondering whether it makes sense to export these into their own variables, and even better perhaps in their own file, something like `src/docs-urls.ts` or something similar.

Doing so would have the benefit of them being more centrally organised (so we can track changed URLs to one place and do a one-move global replace if anything changes).

File: src/autobuild.ts [hunk 2]
```diff
@@ -139,14 +139,14 @@ export async function setupCppAutobuild(codeql: CodeQL, logger: Logger) {
...
-            ? ` To override this, set the ${envVar} environment variable to 'true' in your workflow (see ${envDoc}).`
+            ? ` To override this, set the ${envVar} environment variable to 'true' in your workflow (see ${envDoc} for more information).`
             : ""
         }`,
       );
       core.exportVariable(envVar, "false");
     } else {
       logger.info(
-        `Enabling ${featureName}. This can be disabled by setting the ${envVar} environment variable to 'false' (see ${envDoc}).`,
+        `Enabling ${featureName}. This can be disabled by setting the ${envVar} environment variable to 'false' (see ${envDoc} for more information).`,
```
Comment (NlightNFotis) [inline src/autobuild.ts:149]: In some of these strings we have the URL reference inside of a parenthesis, whereas in others we end the sentence with a period and start a new one.

I'm curious if there are any benefits to being more consistent on that front.

**Review by NlightNFotis — APPROVED** [review]
LGTM broadly, just a couple of nits - don't need to be implemented within the PR.

**Review by henrymercer — DISMISSED** [review]
Nice!  Non-blocking, optional: This could be something we create a CodeQL query for to help make sure we follow this pattern going forward.

File: src/codeql.test.ts (hunk 1 above)
Comment (angelapwen) [reply src/codeql.test.ts]: I did this in an enum, similar to the way we have `EnvVar` for shared environment variables 😄 it was a really good idea — from there I found other instances where we had docs links that ended with periods!

File: src/autobuild.ts (hunk 2 above)
Comment (angelapwen) [reply src/autobuild.ts]: I definitely don't see any downsides to standardizing our docs links — I removed all the parenthetical URL references!

**Comment by angelapwen** [comment]
> Nice! Non-blocking, optional: This could be something we create a CodeQL query for to help make sure we follow this pattern going forward.

Thanks!! I've waited too long to get back to this PR 😆 so I won't write the CodeQL query this time. I think the new `DocUrl` enum out should make it a lot clearer moving forward as well.

**Review by NlightNFotis — APPROVED** [review]

**Pull request merged**